exporter/otlpexporter/README.md

```diff
@@ -15,7 +15,9 @@ The following settings are required:
 
 - `endpoint` (no default): host:port to which the exporter is going to send OTLP trace data,
 using the gRPC protocol. The valid syntax is described
-[here](https://github.com/grpc/grpc/blob/master/doc/naming.md)
+[here](https://github.com/grpc/grpc/blob/master/doc/naming.md).
+Including a scheme of `http` or `https` is allowed in the `endpoint`, but not required.
```

Comment:
Is it recommended, though? As someone reading this, I'm not sure if I should or not include the scheme.

Reply:
Agreed, this was confusing. My intent was to highlight that a scheme of http or https is allowed whereas it was not before. Though it's probably unnecessary since things are pretty well described in the linked gRPC documentation https://github.com/grpc/grpc/blob/master/doc/naming.md, so I removed this sentence.